src/main/java/com/google/sps/servlets/utils/VideoInformation.java

```diff
@@ -24,7 +24,8 @@ public VideoInformation(VideoListResponse videoResponse) {
     VideoStatistics videoStats = video.getStatistics();
     this.videoName = videoSnippet.getTitle();
     this.videoAuthor = videoSnippet.getChannelTitle();
-    this.publishDateString = videoSnippet.getPublishedAt().toStringRfc3339();
+    // Removes the unreadable code at the end of the dateString; Convert to yy-mm-dd format
```

Comment:
This comment is not accurate, this is how you read it: https://tools.ietf.org/html/rfc3339#section-5.8, so they are not unreadable, they are a specific time of the day.
And You are really converting to yyyy-mm-dd(10 digits), please update.